data/records/lhcb-ntuples-hackathon.json

```diff
@@ -0,0 +1,1022 @@
+[
+  {
+    "abstract": {
+      "description": "<p>Data from proton-proton (pp) collisions collected by the LHCb experiment filtered to produce Ntuples for {insert reason for request}.</p><p>FIXME Here should come a long explanation of inputs, ntuples, etc.</p><p>FIXME If we can attach the <code>info.yaml</code> etc, this is the place wher they would be added."
```

Comment:
RE {insert reason for request} -- we should extract this from the request details [here](https://gitlab.cern.ch/cernopendata/lhcb-ntupling-service-requests-dev/-/issues/142) or perhaps an immutable source if it is stored elsewhere

Reply:
I guess this needs to be inputted by the LHCb open data team, as relying on a free text supplied by a user could contain slang, or internal information, or other such language that would not be understandable outside of the context.

(In the reason, 1 user talks to N LHCb open data team; whilst for the promoted ntuples, anybody in the world can access them, missing the original context)